src/usdb_syncer/constants.py

```diff
@@ -164,3 +164,43 @@ class Usdb:
     "Yoruba": "yor",
     "Zulu": "zul",
 }
+
+QUOTATION_MARKS = {
+    # See: https://en.wikipedia.org/wiki/Quotation_mark#Summary_table
+    ("Bosnian", "Finnish", "Swedish"): ("”", "”"),
+    ("Breton", "French"): ("« ", " »"),
+    ("Croatian", "Dutch", "Hebrew", "Hungarian", "Polish", "Romanian"): ("„", "”"),
+    (
+        "Catalan",
+        "Galician",
+        "Greek",
+        "Greek (romanized)",
+        "Italian",
+        "Norwegian",
+        "Portuguese",
+        "Russian",
+        "Russian (romanized)",
+        "Spanish",
+        "Ukrainian",
+    ): ("«", "»"),
+    (
+        "Chinese (romanized)",
+        "English",
+        "Hindi",
+        "Hindi (romanized)",
+        "Indonesian",
+        "Irish",
+        "Korean",
+        "Korean (romanized)",
+        "Portuguese (Brazil)",
+        "Scots",
+        "Turkish",
+        "Vietnamese",
+    ): ("“", "”"),
+    ("Czech", "Estonian", "Icelandic", "Serbian", "Slovak", "Slovenian"): ("„", "“"),
+    ("Danish", "German"): ("»", "«"),
```

Comment:
How did you come up with this list? E.g. why the "alternative" and not the "standard" quotes in German?

Reply:
»Im Amtlichen Regelwerk (z. B. Duden [..], ebenso im Wahrig [...]) werden die Regeln zur Verwendung behandelt, nicht jedoch, ob „deutsche“ Anführungszeichen oder Guillemets zu verwenden sind. Beide Möglichkeiten stehen gleichwertig nebeneinander. [...] Vorteile der Guillemets gegenüber den deutschen Anführungszeichen sind die Unverwechselbarkeit gegenüber Komma, Apostroph und Zollzeichen. Außerdem stören die deutschen Anführungszeichen die Zeilenbildung durch ihr aus der Zeilenmitte gerücktes Bild, die französischen Anführungen hingegen ähneln anderen Interpunktionszeichen nicht und unterstützen die Zeilenbildung. Nach Ansicht des Textportals Rotkel zeichnen sich die Guillemets außerdem durch eine elegantere Erscheinungsform aus.« (https://de.wikipedia.org/wiki/Anf%C3%BChrungszeichen#Deutschland_und_%C3%96sterreich).

In short: aesthetics ;-)